WCON_format.md

```diff
@@ -255,7 +258,7 @@ As an example, inside
       "@XJ":{ "foo": { "e": 2 }, "yes": "I think so"},
       "settings":{"q": 4, "r": 5}
     },
-    "data": { "id":1, "t":0, "x":1, "y":2, "e":3, "@XJ": {"e": 3}, "f":{"q": 4} }
+    "data": [{ "id":1, "t":0, "x":[1], "y":[2], "@XJ": {"e": 3, "f":{"q": 4}}}]
```

Comment:
Getting rid of the ignored `"e"` outside the `"@XJ"` is a good idea, but adding unnecessary brackets around the numbers for `x` and `y` doesn't make this easier to read, nor does making `data` an array when it's not required to be one.  Also, I don't think the text description quite matches any more.